# Request
# (empty)
---
# Response
geometry_msgs/Pose pose